WooCommerce.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -10,6 +10,15 @@
           "version": "4.1.2"
         }
       },
+      {
+        "package": "ScreenObject",
+        "repositoryURL": "https://github.com/Automattic/ScreenObject",
+        "state": {
+          "branch": "trunk",
+          "revision": "68d92f082bf56358f5dba2021e1f5f81107d123e",
+          "version": null
+        }
+      },
```

Comment:
Interesting to see that WooCommerce has the `Package.resolved` file. As I mentioned in the [WordPress PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/16648/files#r647153613) we're not using it in Simplenote.

Looking at it with fresh eyes here. I think tracking it is the appropriate thing to do. This looks to me like the `Podfile.lock` or `package-lock.json` equivalent for the Xcode managed dependency resolution. The Xcode project is the source of truth for the configuration and this is the snapshot of the dependencies as resolved from that configuration that the project expects.

Reply:
> Looking at it with fresh eyes here. I think tracking it is the appropriate thing to do. This looks to me like the Podfile.lock or package-lock.json equivalent for the Xcode managed dependency resolution. The Xcode project is the source of truth for the configuration and this is the snapshot of the dependencies as resolved from that configuration that the project expects.

Correct!